source/_patterns/10-atoms/text/teaser-info.scss

```diff
@@ -0,0 +1,236 @@
+@import '../../variables';
+@import '../../mixins';
+@import '../../breakpoints';
+
+.teaser-info-wrapper {
+  margin: 8px 0 0;
+  padding: 0;
+  display: flex;
+  flex-grow: 1;
+  align-items: flex-end;
+
+  @include tablet-up {
+    margin-top: 12px;
+  }
+}
+
+.teaser-info {
+  position: relative;
+  margin-right: 18px;
+  font-family: $font-regular;
+  font-size: 12px;
```

Comment:
da gibt's dann im Master ein Mixin von Alexej, evtl. noch den Master reinmergen? Dann kann auch das letter-spacing weg und für die Line-height gibt's ziemlich sicher eine Variable

Reply:
Falscher Alarm, ist bereits hier drin: siehe `fontSize` in mixins